docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -137,447 +108,216 @@ If you are using an older version, please upgrade.
             
             Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
             permitted by applicable law.
-            root@ev3dev:~# 
+            robot@ev3dev:~$ 
+
+*   {: tab="Ubuntu"}<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+    This instructions were written using Ubuntu 15.10 using the default desktop
+    (aka Unity) however, they should be similar for other versions of Ubuntu
```

Comment:
It would be a bit easier to read here if the comma were moved to before the "however"